@here/olp-sdk-dataservice-read/test/unit/VersionedLayerClient.test.ts

```diff
@@ -67,17 +69,37 @@ describe("VersionedLayerClient", () => {
             dataServiceRead.RequestFactory,
             "getBaseUrl"
         );
+
         getBaseUrlRequestStub.callsFake(() => Promise.resolve(fakeURL));
     });
 
     afterEach(() => {
         sandbox.restore();
     });
 
-    it("Shoud be initialised", async () => {
+    it("Shoud be initialized", async () => {
         assert.isDefined(versionedLayerClient);
     });
 
+    it("VersionedLayerClient instance with new constructor shoud be initialized with parameters: hrn, layerid, settings and version", async () => {
```

Comment:
VersionedLayerClient instance should be initialized with VersionedLayerClientParams